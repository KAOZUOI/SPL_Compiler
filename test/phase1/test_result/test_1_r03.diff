2c2
< Error type B at Line 6: Missing semicolon ';' LOC1
---
> Error type B at Line 6: Missing semicolon ';'
