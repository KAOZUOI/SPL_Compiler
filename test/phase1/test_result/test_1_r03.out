Error type A at Line 4: unknown lexeme $
Error type B at Line 6: Missing semicolon ';' LOC1
Error type A at Line 8: unknown lexeme @
